### DISCUSSION TITLE
Dates on wrong days of the week

### DISCUSSION THREAD

**Discussion by dustyr** [lukevella/rallly#181] [Q&A]
First I want to say I am really liking Rallly for what I do.  Thanks for all you are doing Luke.

I am still having and issue with some people seeing the dates/days out of synch.  I have my survey set to "Ignore time zones" as the event is in a specific location (Eastern Time).  On my screen and most others the dates shown are Monday June 1, Wednesday June 3, Monday June 8, etc. One of my users, who has a MAC, sent me the attached.  I know it is a bit blurry but it shows on his as Sunday May 31, Tuesday June 2, Sunday  June 7, etc.  All off by one day.  I had him check his date and time zone and they are correct.  I also checked mine which is also correct.  Any ideas?

![image](https://user-images.githubusercontent.com/102229274/169350908-6452ab68-c1d4-4f22-bb24-cf1e0f47de1e.png)

**Comment by lukevella** [comment]
Can't reproduce this myself. Any chance you can email the url for the poll to [support@rallly.co](mailto:support@rallly.co)?
